src/en/hentairead/src/eu/kanade/tachiyomi/extension/en/hentairead/Hentairead.kt

```diff
@@ -185,17 +185,19 @@ class Hentairead : Madara("HentaiRead", "https://hentairead.com", "en", dateForm
         }
     }
 
+    private val chapterExtraDataRegex = Regex("= (\\{[^;]+)")
+    private val pagesDataRegex = Regex(".(ey\\S+).\\s")
```

Comment:
Use `"""` to  avoid escaping inside the regex. Since these are sufficiently complex you should show a cut down example of the intended target text.